README.md

```diff
@@ -25,7 +25,23 @@ This prototype repository covers the various technologies, connect the dots and
 
 ## Installation
 
-We're working hard to bring you a clean & concise guideline for installation. Please check back later.
+1. Duplicate `.env.example` and named it as `.env`.
+2. Add the env variables. Few variables are docker specific and can be skipped.
+    1. [POSTGRES_MULTIPLE_DATABASES, POSTGRES_ROLE]
+3. Run `mix phx.gen.secret` to generate secret and add it to env variable `SECRET_KEY_BASE`
+
+4. Database setup
+    1. Dockerized project setup
+        * `docker-compose config` Check if docker config and env variables are set properly. 
+        * `docker-compose up -d` Start the container.
+    2. Non-dockerized project setup
+        * Create your application database manually.
+
+5. Duplicate `dev.exs.example` and name it as `dev.exs`.
+6. Run `mix setup`.
```

Comment:
is it possible that the `mix` commands here can be a part of DockerFile? and users just need to do `docker-compose up -d` and everything is automatically done?

Reply:
Agreed @rathorevaibhav we are working on it. I've created the tracking here https://github.com/ProjectTides/tides/issues/5